File: takaryo/fizzbuzz.c
#include <stdio.h>
int main(void) {
	int i;
	for (i = 1; i <= 100; i++) {
		printf("%d ",i);
		if (i%3==0) printf("fizz");
		if(i % 5 == 0) printf("buzz");
		printf("\n");
	}
	return 0;
}

File: urano/fizzbuzz.py
#!/usr/bin/env python3

for i in range(1, 114514):
    if i % 15 == 0:
        print(f"{i} fizz buzz")
    elif i % 5 == 0:
        print(f"{i} buzz")
    elif i % 3 == 0:
        print(f"{i} fizz")
    else:
        print(i)
